openverse_catalog/dags/common/loader/reporting.py

```diff
@@ -45,10 +54,13 @@ def report_completion(
     The following data is reported:
         - `duration`: The time the pull data task took to complete. This value is
           "No data" in cases where the pull data task failed.
-        - `cleaned`: The number of records that were removed after the data was loaded
-          into a temporary table. This could be the result of missing columns, or
+        - `missing_columns`: The number of records that were removed after the data was
+          loaded into a temporary table due to missing data in required columns.
           due to multiple records that have the same provider & foreign ID.
```

Comment:
Should this line (59) that was not touched be removed?

Reply:
Oops, great catch!